zsh_completions.go

```diff
@@ -318,7 +318,7 @@ func zshCompGenFlagEntryExtras(f *pflag.Flag) string {
 		case BashCompFilenameExt:
 			extras = ":filename:_files"
 			for _, pattern := range values {
-				extras = extras + fmt.Sprintf(` -g "%s"`, pattern)
+				extras += fmt.Sprintf(` -g "%s"`, pattern)
```

Comment:
Might this work too?

``` golang
extras += ` -g "` + pattern + `"`
```

See https://gist.github.com/dtjm/c6ebc86abe7515c988ec

Reply:
It would, yeah, even though I guess the speed difference is negligible here. We should probably go with whatever we consider more readable.

Reply:
Agree. This is just a matter of style. That's why I approved the PR anyway. I added this comment for the sake of discussion only.

I find avoiding 'sprintf' more readable, because it makes it explicit that no data transformation/representation is implied; this is just a concatenation.

Reply:
I tend to agree, yeah. I'll update the PR.